contrib/hydra/kratos/oidc.github.jsonnet

```diff
@@ -0,0 +1,17 @@
+local claims = {
```

Comment:
OIDC isn't enabled in the config - can these files be removed?

Reply:
Agreed, they can.